blockchain/blocks/src/lib.rs

```diff
@@ -8,11 +8,11 @@ extern crate serde;
 extern crate lazy_static;
 
 mod block;
-mod election_proof;
+pub mod election_proof;
 mod errors;
 pub mod gossip_block;
 pub mod header;
-mod ticket;
+pub mod ticket;
```

Comment:
why re-export? It's exported below?

Reply:
if i dont pub mod it, i cant access the json submodule